rever/activities/conda_forge.xsh

```diff
@@ -227,4 +227,21 @@ class CondaForge(Activity):
 
     def check_func(self):
         """Checks that we can login"""
+        error_on_warn = True
+        index = conda_build.conda_interface.get_index(channel_urls=["conda-forge"])
```

Comment:
hmm don't you have to import `conda_build`?

Reply:
:sheep:

Reply:
And if so, should that be added to the requirements above?

Reply:
hmm, this causes a potentially annoying issue, now this can't be run in a conda env (since conda build needs to be accessed from the root env)

Reply:
maybe we should upstream this to conda smithy?

Reply:
Yeah, that makes sense